server/accounts.go

```diff
@@ -1476,11 +1476,26 @@ func (s *Server) updateAccountClaims(a *Account, ac *jwt.AccountClaims) {
 
 	a.mu.Lock()
 	// Clone to update, only select certain fields.
-	old := &Account{Name: a.Name, imports: a.imports, exports: a.exports, limits: a.limits, signingKeys: a.signingKeys}
+	old := &Account{Name: a.Name, imports: importMap{}, exports: a.exports, limits: a.limits, signingKeys: a.signingKeys}
```

Comment:
We could remove `imports: importMap{}` and just rely on the zero value here, if we wanted to.

Reply:
Good point, will fix.